examples/wind/src/layers/particle-layer/particle-layer.js

```diff
@@ -329,7 +325,7 @@ export default class ParticleLayer extends Layer {
     // This will be a grid of elements
     this.state.numInstances = nx * ny;
 
-    const positions3 = this.calculatePositions3({nx, ny});
```

Comment:
? Don't you need this grid?

Reply:
Now I am using instanced rendering, so I am just rendering a sing instanced point and actual grid offsets are coming from a different instanced buffer (output of TransformFeedback).